abi-types-generator/src/converters/typescript/web3-factory.spec.ts

```diff
@@ -102,11 +102,11 @@ describe('Web3Factory', () => {
   });
 
   describe('buildMethodReturnContext', () => {
-    it('should return `MethodConstantReturnContext<void>` if abiItem.constant === true', () => {
+    it('should return `MethodConstantReturnContext<void>` if abiItem.constant === true  || abiItem.stateMutability === \'view\' || abiItem.stateMutability === \'pure\'', () => {
```

Comment:
can you create this into 3 tests please, 1 for the constant, 1 for the view and 1 for the pure.